apps/chat/src/components/Chat/ChatHeader.tsx

```diff
@@ -156,7 +162,7 @@ export const ChatHeader = ({
                   <ModelIcon
                     entityId={conversation.model.id}
                     entity={model}
-                    size={18}
+                    size={isSmallScreen() ? 20 : 18}
```

Comment:
it makes sense to store `const iconSize = isSmallScreen() ? 20 : 18`  and use this constant in all places